Assets/LoggingManager/WriteToCSV.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using UnityEngine;
+using Debug = UnityEngine.Debug;
```

Comment:
It is because the "Debug" class is present in both *System.Diagnostics* (used for the StopWatches) and *UnityEngine* packages.
In order to use Debug.Log(""), we have to specify from which package to retrieve the function. 
In our case, we are using the Debug function from UnityEngine.